### PR TITLE
Clarify the layer stuff

### PR DESCRIPTION
**Pull request by kalbermattenm**
I do not know if it is clearer now... But in the current state, it was quite obscure for me...

Feel free to merge or close if not ok...

### CODE REVIEW

File: doc/administrator/administrate.rst.mako [hunk 1]
```diff
@@ -56,12 +56,13 @@ Functionalities
...
+In version 2, we split the layer table into 2 tables: ``layer_wms``,
 ``layer_wmts``, and we copy the previous layer table in ``layerv1``.
-There is a tool for migrate the layer from the v1 structure to v2,
-see ``.build/venv/bin/themev1tov2``.
-The ``layerv1`` steal be used in the cgxp application then you should
-maintain the booth version of the layers.
+There is a tool to migrate the layers from the v1 structure to the v2
+structure see ``.build/venv/bin/themev1tov2``.
+If you intend to still use the CGXP application, then you should maintain
+both version of the layers, repesctively ``layerv1`` and
```
Comment (pgiraud) [inline doc/administrator/administrate.rst.mako]: respectively

File: doc/administrator/administrate.rst.mako [hunk 2]
```diff
@@ -56,12 +56,13 @@ Functionalities
...
-In the version 2 we split the layer table in 2 tables: ``layer_wms``,
+In version 2, we split the layer table into 2 tables: ``layer_wms``,
 ``layer_wmts``, and we copy the previous layer table in ``layerv1``.
-There is a tool for migrate the layer from the v1 structure to v2,
-see ``.build/venv/bin/themev1tov2``.
-The ``layerv1`` steal be used in the cgxp application then you should
-maintain the booth version of the layers.
+There is a tool to migrate the layers from the v1 structure to the v2
+structure see ``.build/venv/bin/themev1tov2``.
+If you intend to still use the CGXP application, then you should maintain
```
Comment (pgiraud) [inline doc/administrator/administrate.rst.mako]: Still using the CGXP application? Then you should maintain...

**Comment by kalbermattenm** [comment]
Thanks @pgiraud

**Comment by sbrunner** [comment]
Not related issue, merging :-)

**Comment by sbrunner** [comment]
Thanks @kalbermattenm

**Pull request merged**